storage-shared-benchmarking/src/main/java/com/google/cloud/storage/benchmarking/StorageSharedBenchmarkingCli.java

```diff
@@ -106,15 +106,25 @@ public void run() {
       case "w1r3":
         runWorkload1();
         break;
-      case "w1r3-grpc-dp":
-        runWorkload4();
-        break;
       default:
         throw new IllegalStateException("Specify a workload to run");
     }
   }
 
   private void runWorkload1() {
+    switch (api) {
+      case "JSON":
+        runWorkload1Json();
+        break;
+      case "DirectPath":
```

Comment:
will you add GRPC equivalent which uses CloudPath?

Reply:
eventually thats the idea. for now it just adds flexibility.